src/types.ts

```diff
@@ -0,0 +1,87 @@
+import { GluegunToolbox } from 'gluegun'
```

Comment:
New file where we export all the relevant Ignite CLI types. I really like locating all types in a `src/types.ts` file.